packages/docs-site/astro.config.ts

```diff
@@ -229,17 +236,27 @@ export default defineConfig({
     }),
     robotsTxt({
       sitemap: false,
-      policy: [{
-        userAgent: '*',
-        // disable all crawling, except homepage (for now)
-        allow: '/$',
-        disallow: '/',
-      }],
+      policy: [
+        {
+          userAgent: '*',
+          // The next line enables or disables the crawling on the `robots.txt` level
+          disallow: DMNO_CONFIG.DMNO_ENV !== 'production' ? '/' : '',
+        },
+      ],
     }),
     partytown({
       config: {
         forward: ['dataLayer.push'],
       },
     }),
+    sitemap({
+      filter(page) {
+        return !(
+          page.includes('/deployment/')
+          || page.includes('/nuxtjs')
```

Comment:
we can add `draft` to the frontmatter and all of these will get filtered from the prod build

Reply:
yeah Im pretty sure this plugin just does its own thing